resources/lang/de.json

```diff
@@ -140,10 +140,10 @@
     "events.end": "Ende",
     "events.period": "Zeitraum",
     "events.closestStation": "Nächste Träwelling-Station",
-    "events.on-my-way-to": "Auf dem Weg zu: :name",
-    "events.on-my-way-dropdown": "Ich bin auf dem Weg zu:",
+    "events.on-my-way-to": "Auf dem Weg von/nach: :name",
```

Comment:
Maybe sth along the lines of "Unterwegs wegen: :name" and "Traveling because of: :name"? People can check in if they are going there or going home, I guess.